docs/data/data-grid/filtering/QuickFilteringGrid.tsx

```diff
@@ -35,7 +35,6 @@ function QuickSearchToolbar(props: QuickSearchToolbarProps) {
           endAdornment: (
             <IconButton
               title="Clear"
-              aria-label="Clear"
```

Comment:
I would invert here. Keep `aria-label` but remove `title`. We lose the tooltip but the function of a clear button next to an input is very obvious. The demos for `IconButton` only have `aria-label`: https://mui.com/material-ui/react-button/#icon-button